react/ReadMore/ReadMore.js

```diff
@@ -127,16 +127,15 @@ class ReadMore extends PureComponent<Props, State> {
             <div className={classnames(styles.fadeOut, fadeColor)} />
           ) : null}
         </div>
-        {showMoreLessButton ? (
-          <Button color="transparent" onClick={this.handleShowMore}>
+        {showMoreLessButton && (
+          <TextLink
+            chevron={showMore ? 'up' : 'down'}
+            className={styles.showMore}
+            component={props => <Text {...props} strong />}
```

Comment:
Did you consider making the `Text` component a `button`?

Reply:
Happy to do <a /> tag (shouldn't it be a part of TextLink btw?), button is doing weird things in terms of styles

Reply:
Ah right, it's already in textLink. let's just do a?